### PR TITLE
3659: GDPR contact form

### PR DESCRIPTION
**Pull request by holt83**
#### Link to issue

https://platform.dandigbib.org/issues/3659

#### Description

1. Inserts a hardcoded privacy notice on contact form advicing users not the insert personal data.
2. Remove user cookies set by contact module (se commit message for details).

We can not prevent users from entering personal data, pin etc. and this getting send out unencrypted to various mail servers, but this should make them think twice before hitting that submit button :)

#### Screenshot of the result

3659-contact-form-privacy-notice

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Comment by holt83** [comment]
PR updated to fix scrutinizer errors.

One issue remains which I think we can ignore:

`Only string literals should be passed to t() where possible`

We only pass it through `t()` to make it possible for administrators to change the link and maybe reword the text a bit, and have no need for it to be picked up by extractors.

**Comment by holt83** [comment]
Updated the PR to use the same english source text as in #1626

**Comment by JrgenGNielsen** [comment]
reviewed and approved
